### ISSUE TITLE
Image edit

### ISSUE THREAD

**Issue by winterplayer** [williamyang1991/StyleGANEX#18]
Great job! I have some questions about the code. If I edit images, do I need to do the following? Will removal have an impact on the results?
![微信图片_20231020154958](https://github.com/williamyang1991/StyleGANEX/assets/89123295/cb59af9a-ce9f-4a11-aa6d-f2c119e22a7a)

**Comment by williamyang1991** [comment]
Yes, StyleGANEX is based on StyleGAN, so the face inside your input image needs to be rough rescaled to the size where StyleGAN is pretrained.

**Comment by winterplayer** [comment]
Thanks for the answer.
